test/bitswap.spec.js

```diff
@@ -9,15 +9,16 @@ chai.use(dirtyChai)
 const IPFSApi = require('../src')
 
 const DaemonFactory = require('ipfsd-ctl')
-const df = DaemonFactory.create()
+const df = DaemonFactory.create({remote: true, port: 30003})
```

Comment:
@dryajov I'm confused. From docs seems that this is the way to spawn remove daemons, however, it seems it didn't use it even once for all the tests. Is there any magic trick? Truth be told, tests are failing and so I suspect this to be it.

Reply:
By default, it detects if not running in node, it will assume we're in the browser and use the remote daemon. So, no arguments to `create` will do what you expect depending on env. This was a specific requirement we discussed during the revamp of `ipfsd`.

Reply:
So no need for explicit `remote` unless you have to specify the port.